### DISCUSSION TITLE
Does the license transfer from windows 10 to windows 11?

### DISCUSSION THREAD

**Discussion by NastyaNoTamashii** [AveYo/MediaCreationTool.bat#238] [Q&A]
Will my license from windows 10 be saved to windows 11?

**Comment by AmritP777** [comment]
if upgrading then yes just like a normal windows 10 to 11 upgrade via the windows update. have done this successfully myself. clean install i'm not so sure.

**Comment by MEEMxyz** [comment]
Maybe too late, but I bought a Windows 7 Professional license back in 2009 (full client), and I used it to upgrade to a new installation of Windows 10, and just last week I did a clean install of Windows 11 22H2, and it still recognized this Product Key as valid and continued the installation. So, if your current Windows 10 license is not OEM, it should work with a clean install, if it is OEM, then it is better that you do a "Dynamic Upgrade" as they call the Windows Update one.

**Reply by ericblade** [reply]
Win 7 licenses no longer transfer to 10/11, but any that had been transferred should remain good.  At least, that's my understanding of how it's supposed to work now.
